src/components/editor/PropertyActionButtons.jsx

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+export class PropertyButtonActions extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return(<div className="btn-group" role="group" aria-label="...">
+      <button onClick={this.props.handleMintUri} className="btn btn-success btn-sm">Mint URI</button>
+      <button className="btn btn-default btn-sm" onClick={this.props.handleAddClick}>Add</button>
+    </div>)
+  }
+}
+
+PropertyButtonActions.propTypes = {
+  handleAddClick: PropTypes.func,
+  handleMintUri: PropTypes.func
+}
+
+export default PropertyButtonActions;
```

Comment:
does it matter that this is a file called PropertyActionButtons, but it is declared an exporting PropertyButtonActions?    If it matters, I like PropertyActionButtons better ...

Reply:
and if this *is* a mistake, how is it the tests actually passed?????

Reply:
Good catch! So, it was a mistake and just pushed up a commit that make. Speculating (but haven't confirmed) is that I think when we imported the ProperyActionButtons component in the code and tests, the name of the module was correct and the part of syntax `import ProperyActionButtons` creates a named variable in the calling code.